coordinator/internal/userapi/userapi_test.go

```diff
@@ -354,16 +340,15 @@ func TestRecoveryFlow(t *testing.T) {
 
 	// 1. A Coordinator is created from empty state.
 
-	a := newCoordinator()
+	logger := slog.Default()
+	fs := afero.NewMemMapFs()
+	store := history.NewAferoStore(&afero.Afero{Fs: fs})
+	hist := history.NewWithStore(slog.Default(), store)
+	auth := stateguard.New(hist, prometheus.NewRegistry(), logger)
```

Comment:
I guess we could stub the guard interface here instead of using a real stateguard?